test/getUriParameters-test.coffee

```diff
@@ -255,7 +255,130 @@ describe('Transformation • Refract • getUriParameters' , ->
             "open_now"
           ]
         }]
-      }
+      },
```

Comment:
Since there is no labels in these tests it can be tricky to understand which each one does in the future when someone else comes to this code. Perhaps it would be worth adding a comment describing that this is testing?

```suggestion
      },
      # some comment describing what the following test does
```

Some cases above have some comments such as:

> ```
> # Parameter is optional by default with no type attributes
> ```